index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+  <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js"></script>
+  <script>
+    $(function() {
+      $('[data-toggle="popover"]').popover({
+        trigger: 'focus'
+      }); 
+    })
+  </script>
+  <title>Entry form</title>
+</head>
+<body>
+  <div class="container bg-info">
+    <div class="col-lg-6 offset-lg-4 offset-sm-3 col-sm-6">
+      <h1 class="text-white">
+        <a href="" data-container="body" title="Form" data-trigger="focus" data-toggle="popover" data-placement="bottom" data-content="Quick example to demonstrate Bootstrap’s form styles.">
+          <i class="fas fa-info-circle"></i>
+        </a>
+        This is my form
+      </h1>
+    </div>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="border p-2 rounded">
+        <legend class="w-auto">
+          Personal Information:
+        </legend>
+        <div class="row">
+          <div class="col-lg-8 col-sm-6">
+            <label for="last-name">Surname:</label>
+            <input type="text" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="last-name" placeholder="Фамилия" required>
+          </div>
+          <div class="col-lg-4 col-sm-6">
+            <label for="first-name">Name:</label>
+            <input type="text" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="first-name" placeholder="Имя" required>
+          </div>
+          <div class="col-lg-12 col-sm-12">
+            <label for="input-age">How old are you?:</label>
+            <input type="number" class="form-control shadow-lg p-3 mb-5 bg-white rounded" min="1" max="100" name="years" id="input-age" required>
+          </div>
+          <div class="col-lg-6 offset-lg-2 offset-sm-3 col-sm-6">
```

Comment:
Have a look at the styles applied. Can you see a redundant class in this line?
![screenshot from 2018-12-15 18-56-44](https://user-images.githubusercontent.com/5543497/50296352-043f6080-0483-11e9-8d74-2ebdcaa6ddbd.png)